### DISCUSSION TITLE
vtkPanoramicProjectionPass class in pyvista

### DISCUSSION THREAD

**Discussion by javi-mr** [pyvista/pyvista#3660] [Q&A]
Hi there,

I'm trying to use the method SetProjectionTypeToAzimuthal() of the class vtkPanoramicProjectionPass to get azimuthal projections in Pyvista but I'm not sure how, could you help me?

Thank you!

**Comment by banesullivan** [comment]
Ah just saw this. I posted a comment about this here: https://github.com/pyvista/pyvista/discussions/3582#discussioncomment-4451817
